docs/comparison.rst

```diff
@@ -0,0 +1,93 @@
+
+Comparison with other simulators
```

Comment:
When viewed in rst format on github, Disadvantages for the various simulators are not well formatted into enumerated lists

Reply:
This is an issue on RTD too

Reply:
I still see a problem even with the new updates. I seems to only be a problem in the disadvantages list.